src/components/shared/StepButtons.vue

```diff
@@ -0,0 +1,52 @@
+<template>
```

Comment:
How different are StepButtons than HistoryButtons? Maybe we can use the same component for both cases

Reply:
the main difference is the `$emit` parameter, each has to trigger different actions on the parent component. If we were to use the same component for both we should pass this function as callback prop so that the `$emit` method depends directly on it.

Reply:
and they have different conditions each for disabling user interaction